go.mod

```diff
@@ -61,7 +61,7 @@ require (
 require (
 	github.com/hashicorp/terraform-plugin-framework-timetypes v0.3.0
 	github.com/hashicorp/terraform-plugin-log v0.9.0
-	github.com/stretchr/testify v1.9.0
+	github.com/stretchr/testify v1.10.0
```

Comment:
do we need to upgrade testify as part of these changes?

Reply:
I think it was unintentionally merged in with a newer version of go-tfe earlier. I reverted this.